source/troubleshooting.rst

```diff
@@ -0,0 +1,44 @@
+Troubleshooting
+===============
+
+How do i know what to do?
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Take a look at your logs and at the service, look for any unusual behavior to get a direction on where to look if you don't have one already.
+  Logs are generally stored ``/varl/log/pufferpanel/``
+
+  Service can be checked with ``systemctl status pufferpanel``
+
+
+I am getting permission errors with my files
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+If it does only affect the executable it may be a problem with the template as the steamdownload may require you to set the executable permission, this can be done by adding ``chmod +x <executable>`` to the end of your install process

```

Comment:
If the executable is showing a permission denied error, then it may be a problem